raiden-dapp/src/services/raiden-service.ts

```diff
@@ -83,7 +87,7 @@ export default class RaidenService {
     }
   }
 
-  async connect(subkey?: true) {
+  async connect(stateBackup: string | undefined, subkey?: true) {
```

Comment:
If `stateBackup` is optional one could also do:
```suggestion
  async connect(stateBackup?: string, subkey?: true) {
```